### PR TITLE
Redirect to parent coin when deleting citation/artist

### LINKED ISSUE #2675: Deleting a citation or an artist should return to the parent coin.

### PR DESCRIPTION
**Pull request by christinach**
closes #2675

### CODE REVIEW

File: app/resources/numismatic_artists/numismatic_artists_controller.rb [hunk 1]
```diff
@@ -17,9 +18,18 @@ def after_create_success(obj, change_set)
...
+    redirect_to solr_document_path(@coin)
+  end
+
   private
 
     def parent_resource
       @parent_resource ||= find_resource(@change_set.artist_parent_id)
     end
+
+    def load_coin
```
Comment (escowles) [inline app/resources/numismatic_artists/numismatic_artists_controller.rb]: We eventually want to use this for linking to artists from both Coins and NumismaticIssues, right? Do you want to change the name to `load_parent` or something more neutral?

File: app/resources/numismatic_citations/numismatic_citations_controller.rb [hunk 2]
```diff
@@ -28,4 +34,8 @@ def load_numismatic_references
     def parent_resource
       @parent_resource ||= find_resource(@change_set.citation_parent_id)
     end
+
+    def load_coin
```
Comment (escowles) [inline app/resources/numismatic_citations/numismatic_citations_controller.rb]: Same issue as above, except we already use this from both NumismaticIssue and Coin.

File: app/resources/numismatic_artists/numismatic_artists_controller.rb (hunk 1 above)
Comment (christinach) [reply app/resources/numismatic_artists/numismatic_artists_controller.rb]: I can update to load_parent

File: app/resources/numismatic_artists/numismatic_artist_wayfinder.rb [hunk 3]
```diff
@@ -1,3 +1,4 @@
 # frozen_string_literal: true
 class NumismaticArtistWayfinder < BaseWayfinder
+  inverse_relationship_by_property :parent, property: :numismatic_artist_ids, model: Coin
```
Comment (tpendragon) [inline app/resources/numismatic_artists/numismatic_artist_wayfinder.rb]: Rename this to

```ruby
inverse_relationship_by_property :numismatic_artist_parents, singular: true, property: :numismatic_artist_ids
```

File: app/resources/numismatic_citations/numismatic_citation_wayfinder.rb [hunk 4]
```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
 class NumismaticCitationWayfinder < BaseWayfinder
   relationship_by_property :numismatic_references, property: :numismatic_reference_id, singular: true
+  inverse_relationship_by_property :parent, property: :numismatic_citation_ids, model: Coin
```
Comment (tpendragon) [inline app/resources/numismatic_citations/numismatic_citation_wayfinder.rb]: Rename this to

```ruby
inverse_relationship_by_property :numismatic_citation_parents, singular: true, property: :numismatic_citation_ids
```

**Comment by christinach** [comment]
thanks everyone

**Comment by christinach** [comment]
@escowles can you check please? I did squash the previous commits.

**Pull request merged**